### DISCUSSION TITLE
Social login - Microsoft

### DISCUSSION THREAD

**Discussion by robertlagrant** [ory/.github#21] [Q&A]
I asked this in Slack, but I think it's worth noting here: under Social -> Microsoft, one of the options seems to be an enterprise login:

![image](https://user-images.githubusercontent.com/1766161/188880241-28f8825f-ce72-4602-9218-d7a83ee1ce69.png)

Is that correct? If so, bringing this under "Social" feels a bit of a misnomer.

**Comment by vinckr** [comment]
Yes, I believe that would also include the enterprise login and you can use it for social login for end users but also to connect Azure. I am not familiar with AD at all unfortunately. 

Just from their docs, there is different possibilities to connect on that level and OIDC is just one of them (SAML is coming soon to Kratos btw): 

- https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/auth-oidc

I agree that social login is not the best category here, just all OIDC connections are in that category at the moment. Maybe an "integrations" tab would be more straightforward?
Where would you see this in the docs/look for it in the docs?

**Reply by aeneasr** [reply]
Thank you vincent and good point @robertlagrant - we should probably add terminology around enterprise SSO that includes azure AD as a separate integration in our UI and documentation. @kmherrmann what do you think? Can we get this done?
